java/runtime/src/main/java/org/ray/runtime/gcs/RedisClient.java

```diff
@@ -104,9 +104,9 @@ public boolean exists(byte[] key) {
     }
   }
 
-  public Long incr(byte[] key) {
+  public int incr(byte[] key) {
```

Comment:
this `incr` interface should still return long, because it may be needed by others as well